lib/client/api.go

```diff
@@ -229,16 +229,12 @@ type Config struct {
 	// InsecureSkipVerify is an option to skip HTTPS cert check
 	InsecureSkipVerify bool
 
-	// SkipLocalAuth tells the client not to use its own SSH agent or ask user for passwords. This is
-	// used by external programs linking against Teleport client and obtaining credentials from elsewhere.
-	// e.g. from an identity file.
-	SkipLocalAuth bool
+	// NonInteractive tells the client not to trigger interactive features for non-interactive commands,
+	// such as prompting user to re-login on credential errors. This is used by external programs linking
+	// against Teleport client and obtaining credentials from elsewhere. e.g. from an identity file.
+	NonInteractive bool
```

Comment:
I'm a big fan of this renaming since it's much clearer why this would be used now.

Reply:
+1. Love seeing those big, red blocks in lib/client/client.go as well, way simpler to read.